### PR TITLE
CIV-0000 remove checkForEnableHelmLabel

### PR DESCRIPTION
**Pull request by LeonardoPalmeiro**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by neeta-hmcts — DISMISSED** [review]

**Review by sankhajuria — APPROVED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -99,9 +93,6 @@ withPipeline(type, product, component) {
 
   afterAlways('smoketest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test-results/smokeTest/**/*'
-    if (checkForEnableHelmLabel(env.BRANCH_NAME)) {
```
Comment (sankhajuria) [inline Jenkinsfile_CNP:102]: why are we deleting this ? As we still have to interrupt the build incase of helm enabled

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -60,19 +60,13 @@ def publishRpaContract() {
   sh """ ./gradlew pactPublish """
 }
 
-def checkForEnableHelmLabel(branch_name) {
```
Comment (sankhajuria) [inline Jenkinsfile_CNP:63]: no need to delete it,- we can just change the name of the label

**Review by sankhajuria — CHANGES_REQUESTED** [review]

File: Jenkinsfile_CNP (hunk 2 above)
Comment (LeonardoPalmeiro) [reply Jenkinsfile_CNP]: I agree with you.
I back with this part of the code.

File: Jenkinsfile_CNP (hunk 1 above)
Comment (LeonardoPalmeiro) [reply Jenkinsfile_CNP]: I agree with you.
I back with this part of the code.

**Pull request closed without merging**